lstchain/reco/r0_to_dl1.py

```diff
@@ -182,9 +179,16 @@ def r0_to_dl1(input_filename = get_dataset_path('gamma_test_large.simtel.gz'),
 
     """
     if output_filename is None:
-        output_filename = (
-            'dl1_' + os.path.basename(input_filename).rsplit('.',1)[0] + '.h5'
-        )
+        if (input_filename.startswith('LST')):
+            output_filename = (
+                'dl1_' + os.path.basename(input_filename).split('.',5)[0] + '.' \
```

Comment:
you don't need line continuations inside parenthesis

Reply:
done